pom.xml

```diff
@@ -40,13 +52,32 @@
             <version>2.8.0</version>
         </dependency>
         <dependency>
-            <groupId>org.springframework</groupId>
-            <artifactId>spring-orm</artifactId>
-            <version>5.3.3</version>
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
+            <version>2.5</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>taglibs</groupId>
+            <artifactId>standard</artifactId>
+            <version>1.1.2</version>
+        </dependency>
```

Comment:
redundant dependency

Reply:
![image](https://user-images.githubusercontent.com/65593125/107516920-c26b5a00-6bb5-11eb-8335-9042550b39f6.png)
I don't know why but this depends

Reply:
LEL